### ISSUE TITLE
`npm run lint` fails on a fresh checkout

### ISSUE THREAD

**Issue by zeke** [electron/electron#10593]
Folks contributing PRs with minor docs changes should be able to lint their changes locally without having to bootstrap and/or build electron first.

### Expected behavior

This should work:

```sh
git clone https://github.com/electron/electron
cd electron
npm install
npm run lint
```

### Actual behavior

This error is thrown:

```
> python ./script/cpplint.py

/usr/bin/python: can't open file '/Users/z/Desktop/electron/vendor/depot_tools/cpplint.py': [Errno 2] No such file or directory
```

**Comment by MarshallOfSound** [comment]
If it's just docs changes can we tell them to run `npm run lint-docs`?  Short of adding cpplint to version control theres not much we can do for this

**Comment by MarshallOfSound** [comment]
> Is it part of bootstrap?

Yep 👍

**Comment by jkleinsc** [comment]
Fixed via #10804

**Comment by zeke** [comment]
:)
